config/environments/development.rb

```diff
@@ -51,4 +51,7 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   # config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+
+  # Whitelist web console
+  config.web_console.whitelisted_ips = '10.0.0.0/8'
```

Comment:
This is fine for now, but we may want to consider options to set this outside of the repo with an ENV flag or something.